srcopsmetrics/cli.py

```diff
@@ -98,6 +99,12 @@ def get_entities_as_list(entities_raw: Optional[str]) -> List[str]:
             are stored. Default knowledge path is {StoragePath.DEFAULT.value}
             """,
 )
+@click.option(
```

Comment:
this should be part of the other PR I guess.

Reply:
yes, remvoed